src/components/AudioRecorderTamil/AudioRecorderTamil.js

```diff
@@ -131,6 +131,36 @@ function Mic({
       ],
     });
 
+    function replaceAll(string, search, replace) {
```

Comment:
@vaibhavsTekdi I think this login is already written in result.js. can we use this same method to get the word/sentence result

Reply:
@gouravmore I moved that replaceAll function to utils file & import it on respective "Score" view & Both "VoiceCompare" and "AudioRecorderTamil" components. So that, it can used everywhere by importing it from helper file.

Reply:
@vaibhavsTekdi can we move the other logic also in one place?